doc-content/enterprise-only/openshift/rolemapping-proc.adoc

```diff
@@ -0,0 +1,68 @@
+[id='rolemapping-proc']
+= Providing the LDAP role mapping file
+
+If you configure the `AUTH_ROLE_MAPPER_ROLES_PROPERTIES` parameter, you must provide a file that defines the role mapping. You must mount this file on all affected deployment configurations.
+
+.Procedure
+
+. Create the role mapping properties file, for example, `my-role-map`. The file must contain lines in the following format:
++
+[subs="attributes,verbatim,macros"]
+----
+ldap_role = product_role1, product_role2...
+----
++
+For example:
++
+[subs="attributes,verbatim,macros"]
+----
+admins = kie-server,rest-all,admin
+----
++
+. Create an OpenShift configuration map from the file. Run the following command:
++
+[subs="attributes,verbatim,macros"]
+----
+oc create configmap ldap_role_mapping --from-file=<new_name>=<existing_name>
+----
++
+Where `new_name` is the name that the file is to have on the pods (it must be the same as the name specified in the `AUTH_ROLE_MAPPER_ROLES_PROPERTIES` file) and `existing_name` is the name of the file that you created. For example:
```

Comment:
Suggestion:
Instead of "Where new_name is the name that the file is to have on the pods.."
"Where new_name is the name that the file on the pods..."